includes/group-definitions.inc.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
I think this is getting ahead of yourself.  Possibly remove this until you run into one that needs something more complexity than just the os/group division.